client/js/components/map/admin/AdminLayerListItem.vue

```diff
@@ -30,119 +30,118 @@
       <i
         class="fa fa-bars handle w-[20px] cursor-grab"
         :title="Translator.trans('move')" />
-    </div>
+    </div><!--
+ --><div
+    class="inline-block layout--flush c-at-item__row"
+    data-cy="mapLayerListItem">
     <div
-      class="flex w-full"
-      data-cy="mapLayerListItem">
-      <div class="flex-1">
-        <!-- regular categories -->
-        <i
-          v-if="layer.type === 'GisLayerCategory' && false === layer.attributes.layerWithChildrenHidden"
-          aria-hidden="true"
-          class="fa u-mr-0_125"
-          @click="toggleChildren"
-          :class="[childElements.length > 0 ? (showChildren ? 'fa-folder-open' : 'fa-folder') :'fa-folder-o']" />
-        {{ layer.attributes.name }}
-        <span
-          class="font-size-smaller u-mr-0_5"
-          v-if="isChildOfCategoryThatAppearsAsLayer && 'mapOrder' === sortingType">
+      class="inline-block w-9/12"
+      :class="{ 'w-11/12': !hasPermission('feature_map_layer_visibility') }">
+      <!-- regular categories -->
+      <i
+        v-if="layer.type === 'GisLayerCategory' && false === layer.attributes.layerWithChildrenHidden"
+        aria-hidden="true"
```

Comment:
```suggestion
   
```
The `aria-hidden` attribute should only be applied to non-interactive elements afaik.

Reply:
If I got this right (https://stackoverflow.com/questions/61048356/why-do-we-use-aria-hidden-with-icons)
we should remove the aria-hidden and instead set a aria-label when the icon has no description next to it.
(In occurrences where the icon only decorates a label, the aria-hidden is correct. - as far as I understand it)

Reply:
done here https://github.com/demos-europe/demosplan-core/pull/3368/commits/d1ffb5678ab999e8aaa71f93507553e590b7bc92